exercises/space-age/.meta/generator/space_age_case.rb

```diff
@@ -0,0 +1,12 @@
+require 'generator/exercise_case'
+
+class SpaceAgeCase < Generator::ExerciseCase
+  using Generator::Underscore
```

Comment:
I'm not too familiar with ```refinements```, but it looks like this using statement is necessary. 
It doesn't look like the using statement in ```ExerciseCase``` gets inherited by ```SpaceAgeCase```.